.golangci.yml

```diff
@@ -127,6 +127,7 @@ linters:
     - goerr113
     - gomnd
     - gomoddirectives
+    - govet
```

Comment:
this file is auto-generated, and usually `govet` produces valid errors, I can take a look at them

Reply:
Ah, please do. I couldn't figure out how to solve them...
As for the fields inside the structs, I followed the order used in the SMBIOS specification.
So it probably isn't the most ideal order memory-wise.